docs/user-guide/layer-settings.md

```diff
@@ -134,6 +139,110 @@ The editor is easy to approach thanks also to the following functions:
+
+* Add a **Scale denominator filter** (`max` and `min` scale) within which the level will be displayed on the map by clicking the <img src="../img/button/scale_denominator_button.jpg" class="ms-docbutton"/> button.
+
+* **Remove** the symbolizer by clicking the <img src="../img/button/delete_white_button.jpg" class="ms-docbutton"/> button.
+
+#### Mark
+
+The mark type allows you to add a mark to the layer: clicking on the <img src="../img/button/add_mark_button.jpg" class="ms-docbutton"/> button a mark panel appears: 
+
+<img src="../img/layer-settings/mark_panel.jpg" class="ms-docimage"  style="max-width:500px;">
+
+The mark can have different `Shape`, `Color`, `Stroke` with different `Color` and `Width` and customizable `Radius` and `Rotation`. Take a look at the following example.
+
+<img src="../img/layer-settings/mark_style_ex.jpg" class="ms-docimage">
+
+#### Icon 
+
+With the icon panel, which opens by clicking on <img src="../img/button/add_icon_button.jpg" class="ms-docbutton"/> button, the editor is allowed to add an image as an icon (via its *URL*) and customize the icon `Opacity`, `Size` and `Rotation` angle:
+
+<img src="../img/layer-settings/icon_panel.jpg" class="ms-docimage"  style="max-width:500px;">
+
+#### Line
+
+The line types allows you to customize the lines of the layer: clicking on the <img src="../img/button/add_line_button.jpg" class="ms-docbutton"/> button a line panel appears: 
+
+<img src="../img/layer-settings/line_panel.jpg" class="ms-docimage"  style="max-width:500px;">
+
+The editor can change a `Stroke color`, a `Stroke width`, a `Line style` (*continuous*, *dashed*, etc), the `Line cap` (*Butt*, *Round*, *Square*) and the `Line join` (*Bevel*, *Round*, *Miter*). En example can be the following one:
+
+<img src="../img/layer-settings/ex_line_style.gif" class="ms-docimage">
+
+#### Fill
+
+With the fill panel, which opens by clicking on <img src="../img/button/add_fill_button.jpg" class="ms-docbutton"/> button, the editor is allowed to customize the `Fill color`, the `Outline color` and the `Outline width`:
+
+<img src="../img/layer-settings/ex_fill_style.gif" class="ms-docimage">
+
+#### Text
+
+The text types display a formatted text on a layer: clicking on the <img src="../img/button/add_text_button.jpg" class="ms-docbutton"/> button a text panel appears: 
+
+<img src="../img/layer-settings/text_panel.jpg" class="ms-docimage"  style="max-width:500px;">
+
+The editor is allowed to edit or choose the attribute of the `Label`, customized the `Font` (*DejaVu Sans*, *Serif*, etc , choose the font `Color`, `Size`, `Style` (*Normal* or *Italic*) and `Weight` (*Normal* or *Bold*) and select the desire `Halo color` and `Halo weight`. You can also choose the text `Rotation` and `Offset` (*x* and *y*). En example can be the following one:
```

Comment:
The editor is allowed to type the name of the layer attribute to use for the `Label` and the dropdown list is filtered accordingly to show the existing attributes that are matching the entered test (the user can anyway direclty select an attribute from the list). Moreover, the style editor can customize the `Font Family` (*DejaVu Sans*, *Serif*, etc), choose the font `Color`, `Size`, `Style` (*Normal* or *Italic*) and `Weight` (*Normal* or *Bold*) and select the desired `Halo color` and `Halo weight`. It is also possible to choose the text `Rotation` and `Offset` (*x* and *y*). En example can be the following one: